cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -975,6 +975,8 @@ public static String getZKQuorum(CConfiguration cConf) {
       public static final String PROGRAM_FAILED_RUNS = "program.failed.runs";
       public static final String PROGRAM_KILLED_RUNS = "program.killed.runs";
       public static final String PROGRAM_REJECTED_RUNS = "program.rejected.runs";
+      public static final String PROGRAM_PARTIALLY_GRACEFUL_STOP_RUNS = "program.partially.graceful.stop.runs";
```

Comment:
This namespacing seems odd. I would argue that these 2 metrics are "family".

program.stop.graceful.partially.runs
program.stop.graceful.fully.runs

Seems more in line with namespacing rules

Reply:
Makes sense